package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -625,6 +633,9 @@ export const MessageInputProvider = <
       (prevNumberOfUploads) => prevNumberOfUploads - (pendingAttachments?.length || 0),
     );
     setText('');
+    if (value.editing) {
+      value.clearEditingState();
+    }
```

Comment:
is this a bug fix? previously not having this should have caused a bug right?

Reply:
No, but we send a new message for editing when the message is bounced, that is why it is done as the input didn't reset to normal state in that case. The `resetInput` is the best place to do it so that code is clean hence it is done.